File: Service/model/utilisateur.js
/**
 * Created by Massil on 25/01/2017.
 */
var db = require('./../config/db');

var Schema = db.Schema;

var utilisateur = new Schema({
    nom_utilisateur : 'String',
    mot_de_passe : 'String'
});

module.exports = db.model('Utilisateur' , utilisateur);

File: Suiki/app/src/main/java/com/suiki/suiki/Model/BddModel/Utilisateur.java
package com.suiki.suiki.Model.BddModel;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Massil on 11/01/2017.
 */

public class Utilisateur {
    @SerializedName("nom_utilisateur")
    public String nom_utilisateur ;
    @SerializedName("mot_de_passe")
    public String mot_de_passe ;
    @SerializedName("profile")
    public Personne profile;
}
